drivers/wifi/eswifi/eswifi_socket.c

```diff
@@ -123,7 +123,7 @@ static void eswifi_off_read_work(struct k_work *work)
 		goto done;
```

Comment:
Rewrite your commit title: "drivers: wifi: Fix the use of net_pkt_write() in eswifi driver"